### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by Quintasan**
I have updated (hopefully) a few parts of the website.

Questions remaining:
1. Do we need `pages/faq.md`?
2. I believe that `pages/comprehensive.md` should be reworded or even rewritten. The wording in the table doesn't flow well.
3. I believe that `pages/drop-in-admin.md` could be replaced by a direct link to the Padrino Admin guide or there could be some way to reuse that part in the guide itself.
4. Is Padrino Admin ORM agnostic the same way Padrino is (ie. are ORM adapters available for Padrino available for Padrino Admin as well?)

### CODE REVIEW

**Comment by Quintasan** [comment]
@nesquena Mind taking a look at this too? I'd like to keep working on docs for now until the new site is up (any ETA on that?).

File: padrino-docs/pages/agnostic.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ The available components and their defaults are listed below:
...
 |:--------|:------|:------|
-|orm|none|mongomapper, mongoid, activerecord, datamapper, sequel, couchrest|
-|script|none|prototype, rightjs, jquery, mootools, extcore|
-|renderer|haml|erb, haml|
-|test|rspec|bacon, shoulda, cucumber, testspec, riot, rspec|
-|stylesheet|none|less, sass|
+|orm|none|mongoid, ripple, activerecord, sequel, mongomapper, minirecord, ohm, mongomatic, dynamoid,
+couchrest, datamapper|
+|script|none|rightjs, extcore, dojo, prototype, jquery, mootools|
+|renderer|haml|slim, haml, liquid, erb|
```
Comment (ujifgc) [inline padrino-docs/pages/agnostic.md]: Default renderer is none, liquid should not be here as it's not a renderer.

File: padrino-docs/pages/agnostic.md [hunk 2]
```diff
@@ -11,11 +11,12 @@ The available components and their defaults are listed below:
...
+|test|rspec|rspec, cucumber, minitest, steak, shoulda, riot, bacon|
```
Comment (ujifgc) [inline padrino-docs/pages/agnostic.md]: Default test is none.

File: padrino-docs/pages/drop-in-admin.md [hunk 3]
```diff
@@ -5,20 +5,19 @@ email: nesquena@gmail.com
...
 
 ||
-|Orm Agnostic|Adapters for datamapper, activerecord, sequel, mongomapper, mongoid|
-|Authentication|Account authentication support and permission management|
-|Template Agnostic|View support for Erb and Haml rendering engines|
-|Scaffold|Create a model “admin interface” by invoking a command|
-|MultiLanguage|Translated into 10 languages including English, Spanish, and Italian|
+|Orm Agnostic|Adapters for datamapper, sequel, activerecord, mongomapper, mongoid, couchrest|
+|Authentication|User authentication and authorization management|
+|Template Agnostic|ERB and Haml rendering support|
```
Comment (ujifgc) [inline padrino-docs/pages/drop-in-admin.md]: Slim too.

**Comment by ujifgc** [comment]
> Is Padrino Admin ORM agnostic the same way Padrino is (ie. are ORM adapters available for Padrino available for Padrino Admin as well?)

No. It only understands these:

```
:activerecord 
:minirecord   
:datamapper   
:couchrest    
:mongoid      
:mongomapper  
:sequel       
:ohm          
:dynamoid     
```

File: padrino-docs/pages/agnostic.md (hunk 1 above)
Comment (Quintasan) [inline padrino-docs/pages/agnostic.md]: Isn't the default renderer slim?

**Comment by ujifgc** [comment]
For Padrino 0.12.x default renderer is Slim.
For Padrino ~> 0.13.0 default renderer is None with supported Ruby Stdlib ERB.

**Comment by Quintasan** [comment]
It should be fine now.

**Pull request merged**